egs/wsj/s5/utils/format_lm_sri.sh

```diff
@@ -104,6 +104,27 @@ awk '{print $1}' $out_dir/words.txt > $tmpdir/voc || exit 1;
 change-lm-vocab -vocab $tmpdir/voc -lm $tmpdir/lm.gz -write-lm $tmpdir/out_lm \
   $srilm_opts || exit 1;
 
+# change-lm-vocab introduces error in out_lm when lm.gz has an order
```

Comment:
I think the warning is valid-- after all, that configuration doesn't make
sense, it's effectively a bug in SRILM.  But I want to wait to hear whether
it was just a warning that was the issue, or whether there was later an
actual error.


On Fri, Apr 1, 2016 at 7:55 PM, Kirill Katsnelson <notifications@github.com>
wrote:

> In egs/wsj/s5/utils/format_lm_sri.sh
> <https://github.com/kaldi-asr/kaldi/pull/639#discussion_r58282576>:
>
> > @@ -104,6 +104,27 @@ awk '{print $1}' $out_dir/words.txt > $tmpdir/voc || exit 1;
> >  change-lm-vocab -vocab $tmpdir/voc -lm $tmpdir/lm.gz -write-lm $tmpdir/out_lm \
> >    $srilm_opts || exit 1;
> >
> > +# change-lm-vocab introduces error in out_lm when lm.gz has an order
>
> @danpovey <https://github.com/danpovey>, I can remove the warning. Just
> never thought such a model could be legit, but if it is, maybe the warning
> should not be there at all. What do you think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/639/files/9778cccbca078b8e7e8af1c6ddb7da3b3a1180b8#r58282576>
>